app/models/manageiq/providers/google/network_manager/refresh_parser.rb

```diff
@@ -22,13 +27,98 @@ def ems_inv_to_hashes
         get_security_groups
         get_network_ports
         get_floating_ips
+
+        # Note these load balancer methods must be called in this order
```

Comment:
Don't think this comment is needed, most of the above methods are order dependent (e.g.: `get_instances` has to be after `get_zones` and `get_flavors`)